examples/step-12/step-12.cc

```diff
@@ -458,8 +458,9 @@ namespace Step12
 
   // @sect3{All the rest}
   //
-  // For this simple problem we use the simplest possible solver, called
-  // Richardson iteration, that represents a simple defect correction. This, in
+  // For this simple problem we use a standard iterative solver, called GMRES,
+  // that creates approximate solutions minimising the residual in each
```

Comment:
We tend to use American English :-)
```suggestion
  // that creates approximate solutions minimizing the residual in each
```